From 8aa38333c1f7369795b39261e414903dba463435 Mon Sep 17 00:00:00 2001
From: John Vert <johnv@valvesoftware.com>
Date: Fri, 20 Jan 2017 14:47:55 -0800
Subject: [PATCH 3/7] broadwell enable fast sample c ops

---
 drivers/gpu/drm/i915/intel_pm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index cb950752c346..5813634c2b6a 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -8336,6 +8336,10 @@ static void broadwell_init_clock_gating(struct drm_i915_private *dev_priv)
 	I915_WRITE(CHICKEN_PAR2_1, I915_READ(CHICKEN_PAR2_1)
 		   | KVM_CONFIG_CHANGE_NOTIFICATION_SELECT);
 
+	/* Faster SAMPLE_C operations. */
+	I915_WRITE(HALF_SLICE_CHICKEN3,
+		   _MASKED_BIT_ENABLE(HSW_SAMPLE_C_PERFORMANCE));
+
 	lpt_init_clock_gating(dev_priv);
 
 	/* WaDisableDopClockGating:bdw
-- 
2.14.1

