From e400fdc2e70535725b23e023764db5cd294de834 Mon Sep 17 00:00:00 2001
From: John Vert <johnv@valvesoftware.com>
Date: Fri, 5 May 2017 12:04:21 -0700
Subject: [PATCH 7/7] add -steamos EXTRAVERSION

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 20f7d4de0f1c..20cced8e5711 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 14
 SUBLEVEL = 12
-EXTRAVERSION =
+EXTRAVERSION = -steamos
 NAME = Petit Gorille
 
 # *DOCUMENTATION*
-- 
2.14.1

