From efb674739395630199a062f5be17abcbfbd90aa1 Mon Sep 17 00:00:00 2001
From: John Vert <johnv@valvesoftware.com>
Date: Fri, 5 May 2017 12:04:21 -0700
Subject: [PATCH 7/7] add -steamos EXTRAVERSION

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 14
 SUBLEVEL = 2
-EXTRAVERSION =
+EXTRAVERSION = -steamos
 NAME = Petit Gorille
 
 # *DOCUMENTATION*
