From 5ce1d50c7f153a029c46a9c0e59c5c249290bf28 Mon Sep 17 00:00:00 2001
From: John Vert <johnv@valvesoftware.com>
Date: Fri, 5 May 2017 12:04:21 -0700
Subject: [PATCH 09/10] add -steamos EXTRAVERSION

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 741814d..e2d6cdf 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 4
 PATCHLEVEL = 11
 SUBLEVEL = 4
-EXTRAVERSION =
+EXTRAVERSION = -steamos
 NAME = Fearless Coyote
 
 # *DOCUMENTATION*
-- 
2.7.4

